tests/checks/integrity/new_label_test.py

```diff
@@ -194,6 +194,6 @@ def test_condition_ratio_of_new_label_samples_fail():
 
     assert_that(result, has_items(
         equal_condition_result(is_pass=False,
-                               details='Found new labels [5] in test data, making 25.00% of test data.',
+                               details='Found new labels in test data: [5]\nmaking 25.00% of it samples.',
```

Comment:
why \n? a , would make more sense here

Reply:
If the list is long \n is more convinent IMO

Reply:
maybe adding \n or not should be in the list making code

Reply:
not relevant enough for now IMO